docker/README.md

```diff
@@ -0,0 +1 @@
+#todo
```

Comment:
Noting TODO -- I'd like to document the different tools/images Leo supports here.